package.json

```diff
@@ -2,8 +2,16 @@
   "name": "@loke/logger",
   "version": "3.0.1",
```

Comment:
You gonna bump the version in another PR?

Reply:
yeah